portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportImport.testcase

```diff
@@ -374,286 +434,340 @@ definition {
+		task ("Given: User add a new site and enable local staging") {
+			JSONGroup.addGroup(groupName = "Site Name");
 
-		JSONStaging.enableLocalStaging(groupName = "Site Name");
+			JSONStaging.enableLocalStaging(groupName = "Site Name");
+		}
 
-		WebContentNavigator.openWebContentAdmin(siteURLKey = "site-name-staging");
+		task ("And: Add a new web content on staging then publish it") {
+			WebContentNavigator.openWebContentAdmin(siteURLKey = "site-name-staging");
 
-		WebContentNavigator.gotoAddCP();
+			WebContentNavigator.gotoAddCP();
 
-		WebContent.addCP(
-			webContentContent = "WC WebContent Content",
-			webContentTitle = "WC WebContent Title");
+			WebContent.addCP(
+				webContentContent = "WC WebContent Content",
+				webContentTitle = "WC WebContent Title");
 
-		PortletEntry.publish();
+			PortletEntry.publish();
+		}
 
-		LAR.exportSiteCP(siteScopeName = "Site Name");
+		task ("Then: Site can be exported then published") {
+			LAR.exportSiteCP(siteScopeName = "Site Name");
 
-		Staging.openStagingAdmin(siteURLKey = "site-name-staging");
+			Staging.openStagingAdmin(siteURLKey = "site-name-staging");
 
-		Staging.publishCustomPublication();
+			Staging.publishCustomPublication();
+		}
 	}
 
 	@description = "This is a use case for LPS-95589. Verify is not possible export a DM with deleted document link."
 	@priority = "4"
 	test ExportDMWithDeletedDocumentLink {
 		property test.name.skip.portal.instance = "ExportImport#ExportDMWithDeletedDocumentLink";
 
-		JSONGroup.addGroup(groupName = "Site Name");
+		task ("Given: User add a new site with a new document") {
+			JSONGroup.addGroup(groupName = "Site Name");
 
-		DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "site-name");
+			DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "site-name");
 
-		DMDocument.addCP(
-			dmDocumentDescription = "DM Document Description",
-			dmDocumentFile = "Document_1.jpg",
-			dmDocumentTitle = "DM Document Title");
+			DMDocument.addCP(
+				dmDocumentDescription = "DM Document Description",
+				dmDocumentFile = "Document_1.jpg",
+				dmDocumentTitle = "DM Document Title");
+		}
 
-		DMNavigator.openDocumentTypesAdmin(siteURLKey = "site-name");
+		task ("And: Add a new a document type") {
+			DMNavigator.openDocumentTypesAdmin(siteURLKey = "site-name");
 
-		DMDocumentType.add(
-			dmDocumentTypeDescription = "DM Document Type Description",
-			dmDocumentTypeFieldNames = "Image",
-			dmDocumentTypeName = "DM Document Type Name");
+			DMDocumentType.add(
+				dmDocumentTypeDescription = "DM Document Type Description",
+				dmDocumentTypeFieldNames = "Image",
+				dmDocumentTypeName = "DM Document Type Name");
 
-		DMDocumentType.viewCP(dmDocumentTypeName = "DM Document Type Name");
+			DMDocumentType.viewCP(dmDocumentTypeName = "DM Document Type Name");
+		}
 
-		DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "site-name");
+		task ("When: User add a new a document based on the first document and document type") {
```

Comment:
It's better to be "When: Add a document with the new document type and select the previously created document"